plugins/aws-costs/aws-costs.sh

```diff
@@ -81,6 +83,10 @@ fi
 initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
 final_date_time=$(date -u +"%Y-%m-%d %H:00:00.000")
 
+if [[ "$tagprefix" = "" ]]; then
+  tagprefix = "resource_tags_user_"
+fi
```

Comment:
instead of doing this here, let's do it in the helm chart. That way the user can still unset it.

Reply:
done